src/pd_gmres.m

```diff
@@ -63,11 +63,18 @@
     %             Approximate solution of the linear system.
     %
     %   flag:     boolean
-    %             1 if the algorithm converged, 0 otherwise.
+    %             1 if the algorithm has converged, 0 otherwise.
     %
+    %   relres:   scalar
+    %             Last relative residual norm.
+    %
+    %   resvec:   (1 up to maxit)-by-1 vector
```

Comment:
Is relres equal to last component of resvec?

Reply:
Good point. I believe that in our case, this is true. However, as you pointed out, `gmres` doesn't return the normalised residuals in `resvec`. 

This is a bit confusing to be honest, shall we rename `resvec` to `relresvec`?

Reply:
I think is  more descriptive 'relresvec

Reply:
Great. I will incorporate the change. This also means that when the unrestarted block is used, we need to normalise the `relres` output of `gmres` to create the `relresvec` (which in this case will always be a `2-by-1` vector). Right?

See, in particular: https://github.com/nidtec-una/krysbas-dev/blob/c9ee5a8697c58746307e2071e00a4787525a96d8/src/pd_gmres.m#L264

Reply:
@jccf19 : I know you already approved the changes, but I will request you do a quick code review once again, since a lot of changes were included after our conversation. Thanks!

Reply:
Please resolve this conversation if you agree with the changes :)